src/routes/AppRoutes.tsx

```
@@ -48,6 +48,38 @@ const MyProjectVolunteersPass = lazy(
 );
+const MyNotifications = lazy(
+  () => import('../components/mypage/notifications/Notifications')
+);
+const NotificationsAll = lazy(
+  () => import('../components/mypage/notifications/all/All')
+);
+const NotificationsAppliedProjects = lazy(
+  () =>
+    import('../components/mypage/notifications/appliedProjects/AppliedProjects')
+);
+const NotificationsCheckedApplicants = lazy(
```

ESLint

[error] 'NotificationsCheckedApplicants' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)